src/sentry/web/frontend/vercel_extension_configuration.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+
+from sentry.integrations.pipeline import IntegrationPipeline
+from sentry.web.frontend.base import BaseView
+
+
+class VercelExtensionConfigurationView(BaseView):
+    auth_required = False
+
+    def get(self, request, *args, **kwargs):
```

Comment:
actually one last thing - we should probs feature gate this view (not just the `OrganizationConfigIntegrationsEndpoint`) since you can start in Vercel. I realize rn its not really that much of an issue because we don't have the vercel integration published yet or anything but still worth doing